Misc/NEWS.d/next/Core and Builtins/2024-04-02-17-37-35.gh-issue-117431.vDKAOn.rst

```diff
@@ -0,0 +1,9 @@
+Improve the performance of the following :class:`str` methods
+by adapting them to the :c:macro:`METH_FASTCALL` calling convention:
+
+* :meth:`~str.endswith`
+* :meth:`~str.find`
+* :meth:`~str.index`
+* :meth:`~str.rfind`
+* :meth:`~str.rindex`
+* :meth:`~str.startswith`
```

Comment:
This PR also contains a changes related to `str.count`. Is it worth mentioning this in a `NEWS` entry?

Reply:
Yes, thanks for the catch!